manim/mobject/types/vectorized_mobject.py

```diff
@@ -846,11 +846,30 @@ def get_subcurve(self, a, b):
 
 class VGroup(VMobject):
     def __init__(self, *vmobjects, **kwargs):
-        if not all([isinstance(m, VMobject) for m in vmobjects]):
-            raise Exception("All submobjects must be of type VMobject")
         VMobject.__init__(self, **kwargs)
         self.add(*vmobjects)
 
+    def add(self, *vmobjects):
+        """Checks if all passed elements are an instance of VMobject and then add them to submobjects
+
+        Parameters
+        ----------
+        vmobjects :
+            List of VMobject to add
+
+        Returns
+        -------
+        None
+
+        Raises
+        ------
+        Exception
+            If one element of the list is not an instance of VMobject
+        """
+        if not all([isinstance(m, VMobject) for m in vmobjects]):
+            raise Exception("All submobjects must be of type VMobject")
```

Comment:
Suggestion: while we're at it, we could change this `Exception` to a `TypeError`. See [here](https://docs.python.org/3/library/exceptions.html#TypeError).